docs/learn/learn-staking.md

```diff
@@ -6,110 +6,324 @@ description: An introduction on staking in Polkadot's NPoS consensus model.
+
+### Selection of validators
+
+Choosing validators is not a simple task and it should take into account for nominator reward and
+risk preferences. Ideally one aims to maximize the reward-to-risk ratio by maximizing rewards and minimizing risks, with sometimes having to
+compromise between the two as minimizing risks might decrease rewards as well. Nominators should pay attention especially to six criteria when nominating validators (not in order of importance):
+
+- amount of era points
+- validator pool, equivalent to own stake (i.e. coming from the validator) + other stake (i.e. coming from nominators)
+- own stake
+- commission fees (i.e. how much validators charge nominators)
+- verified identity
+- previous slashes
+
+The diagram below shows how the selection of those criteria affects reward-to-risk ratio.
+
+![rewards and risks diagram](../assets/staking/Reward-risk%20nominating.png)
+
+#### In theory
+
+To maximize rewards and minimize risk one could select those validators
+that:
+
+- have era points above average (because they will get more rewards for being active), 
+- have validator pool below average (because they will pay out more rewards per staked {{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }}), 
+- have high own stake (because if slashed they have a lot to lose), 
+- have low commission fees but not 0% (because it makes sense that for doing the heavy lifting validators ask for a small commission), 
+- have on-chain registered identity (because increases trust and you can reach out to them or look at their website or social media), 
+- and have not been slashed (meaning that their on-chain behavior is genuine).
+
+#### In practice
+
+:::caution Nominators must periodically check their validators
+
+Nominating is _not_ a "set and forget" operation. The whole NPoS system is dynamic and as such nominators should
+periodically monitor the performance and reputation of their validators. Failing to do so could
+result in applied slashes and/or rewards not being paid out, even for a prolonged period of time.
+
+:::
+
+Although the theory can be used as a general guideline, the practice is more complicated and
+following the theory might not necessarily lead to the desired result. Validators might have validator pool below average, low commission and above average era points in one era and then having a totally
+different profile in the next one. There are criteria that vary more than others with era points
+being the most variable and thus the probabilistic component of staking rewards. It is not recommended
+to change nominations at each era only because the era points of one validator were low. Selection
+based on identity, no slashes and low commission should be fine. Variability in rewards due to the
+other criteria should level out over time. There might be the case when one validator always gets
+below average era points, this is a case when selecting another validator might help to increase the
+stake performance.
```

Comment:
Rephrasing this paragraph. If I modified the original intent/ideas presented, feel free to make updates.

```suggestion
different profile in the next one. Selection based the criteria like on-chain identity, slash history and low commission make the staking rewards deterministic. But there are criteria that vary more than others, with era points
being the most variable and thus one of the key probabilistic components of staking rewards. It is not recommended
to change nominations because of the low era points of a validator in a single era. Variability in rewards due to the era points should level out over time. If a validator consistently gets
era points below average, it makes sense to nominate a better performing validator for the health of the network and for increased staking rewards.
```